gtfs/spec/en/reference.md

```diff
@@ -514,22 +514,22 @@ For a given ordered pair of arriving trip and departing trip, the transfer with
 
 |  Field Name | Type | Presence | Description |
 |  ------ | ------ | ------ | ------ |
-|  `from_stop_id` | Foreign ID referencing `stops.stop_id` | **Conditionally Required** | Identifies a stop or station where a connection between routes begins. If this field refers to a station, the transfer rule applies to all its child stops. Refering to a station is forbiden for `transfer_types` 4 and 5. |
-|  `to_stop_id` | Foreign ID referencing `stops.stop_id` | **Conditionally Required** | Identifies a stop or station where a connection between routes ends. If this field refers to a station, the transfer rule applies to all child stops. Refering to a station is forbiden for `transfer_types` 4 and 5. |
+|  `from_stop_id` | Foreign ID referencing `stops.stop_id` | **Required** | Identifies a stop or station where a connection between routes begins. If this field refers to a station, the transfer rule applies to all its child stops. Refering to a station is forbiden if `transfer_type` is `4`. |
```

Comment:
Can we update `Refering` -> `Referring` and `forbiden` -> `forbidden`

Reply:
I think we should explicitly mention, in both `from_stop_id` and `to_stop_id`: 
Stops with `location_type` values `2` (Entrance/Exit), `3` (Generic Node), or `4` (Boarding Area) are forbidden. 
This would justify the presence of the validator rule [`transfer_with_invalid_stop_location_type`](https://github.com/MobilityData/gtfs-validator/blob/master/RULES.md#transfer_with_invalid_stop_location_type).